dom.bs

```diff
@@ -3594,6 +3625,60 @@ run these steps:
 <ol>
  <li><p>Assert: either <var>addedNodes</var> or <var>removedNodes</var> <a for=set>is not empty</a>.
 
+ <li><p>If either <var>options</var>'s' {{MutationObserverInit/elementLocalNameFilter}} or
+ {{MutationObserverInit/elementFilterByAttribute}} is present, then:
+
+ <ol>
+  <li><p>Let <var>nodeAttributesList</var> be the set of local names of node's attributes.
+
+  <li><p>Let <var>filteredAddedNodes</var> be an empty <a for=/>list</a>.
+
+  <li><p>Let <var>filterRemovedNodes</var> be an empty <a for=/>list</a>.
+
+  <li>
+   <p>For each <var>node</var> in <var>addedNodes</var></a>.
+
+   <li>
```

Comment:
I would expect a for each to be followed by a nested list so there's an `<ol>` missing here I think.